validphys2/src/validphys/tests/test_sumrules.py

```diff
@@ -0,0 +1,55 @@
+"""
+    Test for the sum rules
+"""
+import pytest
+import pandas as pd
+from validphys.api import API
+from validphys.tableloader import sane_load
+
+from .conftest import PDF, HESSIAN_PDF
+from .test_regressions import make_table_comp, parse_data_cv
+
+Q = 10  # GeV
+
+
+def parse_sumrules(filename):
+    """Parse the regression"""
+    return sane_load(filename, header=0)
+
+
+@pytest.mark.parametrize("pdf_name", [PDF, HESSIAN_PDF])
+def test_central_value(pdf_name):
+    """Check that the central value is the same as the mean"""
+    all_sr = API.sum_rules_table(pdf=pdf_name, Q=Q)
+    cv_sr = API.central_sum_rules_table(pdf=pdf_name, Q=Q)
+    pd.testing.assert_series_equal(
+        cv_sr.squeeze(), all_sr["mean"], atol=1e-5, check_names=False
+    )
+
+
+def _regression_sum_rules(pdf_name):
+    known_sumrules = API.sum_rules_table(pdf=pdf_name, Q=Q)
+    central_val = API.central_sum_rules_table(pdf=pdf_name, Q=Q)
+    # First concatenate the central value to the sum rules
+    ret = pd.concat([known_sumrules, central_val], axis=1)
+    # The unknown sum rules need to be treated a bit because
+    # 1) They don't contain min or max
+    # 2) They are of type object?
```

Comment:
This is so that the central value and error are displayed with a reasonable number of decimals in all cases. See

https://github.com/NNPDF/reportengine/blob/675036a93ffc853398d169501d7fc40c9c2cb168/src/reportengine/floatformatting.py#L55

I guess one could use `unknown_sum_rules` action instead.

Reply:
I see. But the other problem stays so not sure if it changes much (and the others are floats)